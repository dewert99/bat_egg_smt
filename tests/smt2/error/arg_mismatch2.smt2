(declare-fun f (Bool Bool) Bool)
(assert (f true))