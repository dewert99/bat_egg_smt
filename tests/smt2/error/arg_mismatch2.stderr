the function `f` expects 2 arguments but got 1 at 2:12
(assert (f true))
           ^

